web-wallet/src/lib/containers/StakeContract/StakeContract.svelte

```diff
@@ -117,6 +110,7 @@
 		gasSettings,
 		language
 	] = collectSettings($settingsStore);
+	const { hideStakingNotice } = $settingsStore;
```

Comment:
Unless you did it on purpose to avoid reactivity, you can just add `getKey("hideStakingNotice")` to the array passed to `collect` in the `collectSettings` definition and you can:
```js
$: [
    gasSettings,
    language,
    hideStakingNotice
] = collectSettings($settingsStore);
```

Reply:
Just tried that, it seems it's not working.

https://github.com/dusk-network/rusk/assets/34094428/a864ea1e-4d17-42f4-8af6-0261aa50bbfd

The reason I did it as I did was because I'm not too sure I understand how the `collectSettings` works. It looks to me as it's a function on the language and the language related properties, which the hide notice isn't. 🤷‍♂️

Reply:
We needed it to be not reactive.